StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/IntentConfirmParams.swift

```diff
@@ -26,6 +26,7 @@ class IntentConfirmParams {
 
     let paymentMethodParams: STPPaymentMethodParams
     let paymentMethodType: PaymentSheet.PaymentMethodType
+    let paymentMethodOptions: STPConfirmPaymentMethodOptions
```

Comment:
Nit: Let's maybe rename to `confirmPaymentMethodOptions` for naming consistency with other two?